src/test/scala/io/iohk/ethereum/jsonrpc/EthProofServiceSpec.scala

```diff
@@ -95,11 +96,22 @@ class EthProofServiceSpec
     result.isRight shouldBe true
     result.fold(
       l => l,
-      r =>
-        r.proofAccount.storageProof.map(v => {
+      r => {
+        val accountProof = r.proofAccount
```

Comment:
there could be, but I would make it explicit for each test. In my opinion it makes it more readable and easier in case we change some input values in the future